diff -Nur -x '*.orig' -x '*.rej' e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS mezzanine_patched_e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS
--- e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS	2007-06-01 03:36:58.000000000 -0600
+++ mezzanine_patched_e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS	2007-06-01 03:30:17.000000000 -0600
@@ -1,10 +1,10 @@
 {
     my $model = $nut{Model} || "newhidups";
     my $device = $nut{Device} || "/var/lib/ups/hiddev0";
-    my $type = $nut{Type} || '';
+    my $type = $nut{Type};
     $OUT .= "[UPS]\n";
     $OUT .= "\tdriver = $model\n";
     if  ($model eq 'genericups')
-    { $OUT .= "\tupstype = $type\n" if $type; }
+    { $OUT .= "\tupstype = $type\n" if defined $type; }
     $OUT .= "\tport = $device\n";
 }
