diff -Nur -x '*.orig' -x '*.rej' e-smith-nutUPS-1.2.0/createlinks mezzanine_patched_e-smith-nutUPS-1.2.0/createlinks
--- e-smith-nutUPS-1.2.0/createlinks	2006-03-15 12:51:21.000000000 -0700
+++ mezzanine_patched_e-smith-nutUPS-1.2.0/createlinks	2007-01-19 10:03:31.000000000 -0700
@@ -6,7 +6,12 @@
 
 templates2events("/etc/sysconfig/ups", @events);
 
-foreach (qw(ups.conf upsd.conf upsd.users upsmon.conf upssched.conf))
+foreach (qw(ups.conf upsd.users upsmon.conf upssched.conf))
 {
     templates2events("/etc/ups/$_", @events);
 }
+
+foreach (qw(bootstrap-console-save console-save))
+{
+    templates2events("/etc/ups/upsd.conf", $_);
+}
