diff -Nur -x '*.orig' -x '*.rej' e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS mezzanine_patched_e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS
--- e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS	2006-03-15 12:51:21.000000000 -0700
+++ mezzanine_patched_e-smith-nutUPS-1.2.0/root/etc/e-smith/templates/etc/ups/ups.conf/UPS	2007-01-13 10:46:04.000000000 -0700
@@ -1,7 +1,9 @@
 {
     my $model = $nut{Model} || "newhidups";
     my $device = $nut{Device} || "/var/lib/ups/hiddev0";
+    my $type = $nut{Type} || '';
     $OUT .= "[UPS]\n";
     $OUT .= "\tdriver = $model\n";
+    $OUT .= "\tupstype = $type\n" if $type;
     $OUT .= "\tport = $device\n";
 }
