diff -Nur -x '*.orig' -x '*.rej' e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/status mezzanine_patched_e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/status
--- e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/status	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/status	2005-07-20 18:04:45.985838184 -0600
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/type mezzanine_patched_e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/type
--- e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/type	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-nutUPS-1.1.0/root/etc/e-smith/db/configuration/defaults/nut/type	2005-07-20 18:04:42.131424144 -0600
@@ -0,0 +1 @@
+service
diff -Nur -x '*.orig' -x '*.rej' e-smith-nutUPS-1.1.0/root/etc/e-smith/etc/usb/usb.usermap/00header mezzanine_patched_e-smith-nutUPS-1.1.0/root/etc/e-smith/etc/usb/usb.usermap/00header
--- e-smith-nutUPS-1.1.0/root/etc/e-smith/etc/usb/usb.usermap/00header	2002-11-18 15:56:32.000000000 -0700
+++ mezzanine_patched_e-smith-nutUPS-1.1.0/root/etc/e-smith/etc/usb/usb.usermap/00header	1969-12-31 17:00:00.000000000 -0700
@@ -1,2 +0,0 @@
-# usb module         match_flags idVendor idProduct bcdDevice_lo bcdDevice_hi bDeviceClass bDeviceSubClass bDeviceProtocol bInterfaceClass bInterfaceSubClass bInterfaceProtocol driver_info
-
